angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -87,15 +87,16 @@
 
       <!-- For basic apps (not a widget, not simple content, no pithy content), show only an icon -->
       <div ng-switch-when="NORMAL">
-        <a tabindex="-1" ng-href="{{widgetCtrl.renderURL(portlet)}}" target="{{portlet.target}}" class="normal-widget">
+        <a tabindex="-1" ng-href="{{ widgetCtrl.renderURL(portlet) }}" target="{{portlet.target}}" class="normal-widget">
           <div class="widget-icon-container" layout="column" layout-align="center center">
             <portlet-icon></portlet-icon>
           </div>
         </a>
-        <button aria-labelledby="goToApps-{{portlet.nodeId}} appTitle_portlet.title-{{portlet.nodeId}}" class="launch-app-button">
-          <span ng-if='portlet.widgetConfig.launchText'>{{portlet.widgetConfig.launchText}}</span>
-          <span ng-if='!portlet.widgetConfig.launchText'>Launch full app</span>
-        </button>
+        <launch-button data-href="{{ widgetCtrl.renderURL(portlet) }}"
+                       data-target="_blank"
+                       data-button-text="{{ portlet.widgetConfig.launchText ? portlet.widgetConfig.launchText : 'Launch full app' }}"
+                       data-aria-label="goToApps-{{portlet.nodeId}} appTitle_portlet.title-{{portlet.nodeId}}">
```

Comment:
I think `data-aria-label` is [intended to be a human-ready label](http://uw-madison-doit.github.io/uw-frame/latest/#/md/directives), but it looks like identifiers here?